_posts/2020-06-17-ombulabs-open-source-guidelines.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "OmbuLabs Open Source Guidelines"
+date: 2020-06-17 11:16:00
+categories: ["open-source", "guideline"]
+author: bronzdoc
+---
+
+![goHbayOl3PPzk7yYgEEOzF8q9zqRurIwZexM1ge8.jpg](https://tettra-production.s3.us-west-2.amazonaws.com/0d6efb4f154041e899af17bdcd19c1b5/ce8d5f452da02a88554d80c256162a35/d822b155a4112474fdb7aea5ee22465e/d28647d6b4d6e5842508679707ee21a7/goHbayOl3PPzk7yYgEEOzF8q9zqRurIwZexM1ge8.jpg)
+
+At OmbuLabs we dedicate some of our work hours to open source every week. Also some of us contribute to open source during our spare time.
+
+Here are a few tips to that help us keep OSS contributions organized
+
+<!--more-->
+
+From now on every-time we contribute to an OSS project in working hours, we fork it to the [FastRuby.io organization](https://github.com/fastruby).
+
+This way we can keep track of the contributions we made as a company, also the branches created in the project are accessible to the rest of the team.
+
+As a company we get these benefits:
+
+\- We use OSS projects and we want them to be better
+
+\- We want our brand to be associated with these OSS projects
```

Comment:
since the title of this list is `we get these benefits`, for me reading `we want...` sounds weird, how about something like `Our brand can be associated with these OSS projects`, `Our teammates can learn more about these projects`, etc?

Reply:
You tagged another user lol

Reply:
You mean to remove all the `we want` ... ? form all the benefits?

Reply:
oops hahaha sorry other luis xD

What I mean is changing "we want something" with "we can do something", like stating that's something we can do now and not something we want to do (sounds more like a desire). It's just me being picky though hahah.